resources/assets/js/app/components/MainNavigation.js

```diff
@@ -29,6 +29,8 @@ class MainNavigation extends Component {
     render() {
         const navClass = !this.state.isNavOpen ? 'collapse' : '';
         const dropdownClass = this.state.isDropdownOpen ? 'show' : '';
+        const logInUrl = !!sessionStorage.jwt ? '/logout' : '/login';
```

Comment:
all work with session should be in AuthManager/AuthService, it should know how to verify if session active or not